### DISCUSSION TITLE
if there's a bug, check it pls

### DISCUSSION THREAD

**Discussion by kisekiremi** [community/community#64802] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

i use the "ctrl+f" to search the code in the Files's interface, but while i click the "ctrl+f"'s next btn & scroll the page, the code are stocking together and a new node is created on the DOM tree's tail.

![screenshot](https://github.com/community/community/assets/29427097/1a95f589-fc9f-4e7c-a8df-69bf473d8232)

**Comment by naserykarim922** [comment]
Install the Pinterest tool
